packages/waku/src/router/client.ts

```diff
@@ -81,6 +81,13 @@ const parseRouteFromLocation = (): RouteProps => {
   return parseRoute(new URL(window.location.href));
 };
 
+function isAltClick(event: MouseEvent<HTMLAnchorElement>) {
```

Comment:
Can you make it an arrow function? (Just a preference)

Reply:
All set